posthog/tasks/stop_surveys_reached_target.py

```diff
@@ -0,0 +1,47 @@
+from typing import List, Dict
```

Comment:
Seems like the formatter wasn't applied here, also I doubt the the types are correct in this file. I'll fix it soon